package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "hyperapp",
   "description": "1kb JavaScript library for building frontend applications.",
-  "version": "0.9.2",
+  "version": "0.9.3",
```

Comment:
Whoops. My origin points to my fork of hyperapp so the last release actually updated my fork instead. 🙇